drivers/pca9685/pca9685.c

```diff
@@ -274,21 +273,21 @@ static int _init(pca9685_t *dev)
 
     /* set Auto-Increment flag */
     uint8_t byte = 0;
-    _set_reg_bit(&byte, PCA9685_MODE2_INVERT, dev->params.inv);
-    _set_reg_bit(&byte, PCA9685_MODE2_OUTDRV, dev->params.out_drv);
-    _set_reg_bit(&byte, PCA9685_MODE2_OUTNE, dev->params.out_ne);
+    bitarithm_set_masked(&byte, PCA9685_MODE2_INVERT, dev->params.inv);
```

Comment:
I would prefer to make these changes within a separate PR once this PR is merged.

Reply:
Will do, just thought it's easier when I provide an example with it.